docs/fast_data/inputs_and_outputs.md

```diff
@@ -1225,6 +1228,214 @@ Example:
 </p>
 </details>
 
+#### Message format v2.0.0
+
+This version of Single View Update event is emitted when service environment variable `SV_UPDATE_VERSION` is set to `v2.0.0`.
+In future major releases, this will become the default message format.
+
```

Comment:
Could be nice to mention here that sv-update and pr-update (both 2.0.0) are fully compatible so the external gathering mechanisms can be simplified

Reply:
I would add from which version of the service this env var could be used and from which version of the service the sv-update v2.0.0 can be exploited. I agree with Ian that we can add the fact that the 2 messages are compatible

Reply:
I updated the doc